evalml/automl/automl_search.py

```diff
@@ -1262,11 +1261,14 @@ def full_rankings(self):
             "high_variance_cv",
             "parameters",
         ]
+        full_rankings_cols = pipeline_results_cols + ["search_order"]
+
         if not self._results["pipeline_results"]:
             return pd.DataFrame(columns=full_rankings_cols)
 
         rankings_df = pd.DataFrame(self._results["pipeline_results"].values())
-        rankings_df = rankings_df[full_rankings_cols]
+        rankings_df = rankings_df[pipeline_results_cols]
+        rankings_df["search_order"] = pd.Series(self._results["search_order"])
```

Comment:
Does this mean search order is the last column? Might make more sense to put it next to the pipeline name?

Reply:
Yeah :'( we get the other info from pipeline_results so I didn't want to break that up, but I think this makes more sense too